Samples/StreamRecorder/StreamRecorderApp/RMCameraReader.cpp

```diff
@@ -102,6 +105,45 @@ void RMCameraReader::CameraUpdateThread(RMCameraReader* pCameraReader, HANDLE ca
     }
 }
 
+void SaveAccel(IResearchModeSensorFrame* pSensorFrame, IResearchModeAccelFrame* pSensorAccelFrame)
+{
+    DirectX::XMFLOAT3 sample;
+    char printString[1000];
+    HRESULT hr = S_OK;
+
+    ResearchModeSensorTimestamp timeStamp;
+    UINT64 lastSocTickDelta = 0;
+    UINT64 glastSocTick = 1;
+
+    pSensorFrame->GetTimeStamp(&timeStamp);
+
+    if (glastSocTick != 0)
+    {
+        lastSocTickDelta = timeStamp.HostTicks - glastSocTick;
+    }
+    glastSocTick = timeStamp.HostTicks;
+
+    hr = pSensorFrame->QueryInterface(IID_PPV_ARGS(&pSensorAccelFrame));
+    if (SUCCEEDED(hr))
+    {
+        hr = pSensorAccelFrame->GetCalibratedAccelaration(&sample);
+        if (FAILED(hr))
+        {
+            return;
+        }
+        sprintf(printString, "####Accel: % 3.4f % 3.4f % 3.4f %f %I64d %I64d\n",
+            sample.x,
+            sample.y,
+            sample.z,
+            sqrt(sample.x * sample.x + sample.y * sample.y + sample.z * sample.z),
+            (lastSocTickDelta * 1000) / timeStamp.HostTicksPerSecond
+        );
```

Comment:
Hier berechnest du deine IMU Daten und schreibst sie als String in die Variable printString. printString verwendest du nie wieder...